app/templates/components/forms/events/view/edit-speaker.hbs

```diff
@@ -0,0 +1,46 @@
+<form class="ui form {{if isLoading 'loading'}}" {{action 'submit' on='submit'}}>
+  <div class="field">
```

Comment:
The input fields should be based on the custom form settings for that specific event. Must not be hardcoded.

Reply:
@ritikamotwani You can check how it is implemented in `events/id/sessions/create`